modules/nlu/src/backend/intents/entities-featurizer.test.ts

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash'
+
+import Utterance, { UtteranceEntity, UtteranceRange } from '../utterance/utterance'
+
+import { getEntitiesEncoding } from './entities-featurizer'
+
+type Entity = UtteranceRange & UtteranceEntity
+function buildEntity(type: string): Entity {
+  return {
+    confidence: 0,
+    endPos: 0,
+    endTokenIdx: 0,
+    metadata: {
+      entityId: 'The Mordor',
+      extractor: 'list',
+      source: 'Bilbo baggins'
+    },
+    startPos: 0,
+    startTokenIdx: 0,
+    type,
+    value: 'frodo the hobbit',
+    sensitive: false
+  }
+}
+
+class UtteranceMock extends Utterance {
```

Comment:
There's a test utils for that. 

check in utterance.ts There's something like `makeTestUtterance`